libavcodec/cbs_h2645.c

```diff
@@ -2199,6 +2199,12 @@ static const SEIMessageTypeDescriptor cbs_sei_common_types[] = {
         sizeof(SEIRawDisplayOrientation),
         SEI_MESSAGE_RW(sei, display_orientation)
     },
+    {
+        SEI_TYPE_FRAME_FIELD_INFO,
```

Comment:
this is not a common  sei.  need to put it H274

frame_field_info( payloadSize ) {
    ffinfo_pic_struct u(4)
    ffinfo_source_scan_type u(2)
    ffinfo_duplicate_flag u(1)
}